examples/Kentico.Xperience.UMT.Examples/SampleProvider.cs

```diff
@@ -120,6 +122,8 @@ public static List<IUmtModel> GetFullSample()
             ContentItemSamples.SampleArticleDataEnGbWithRelationsReference,
             ContentItemSamples.SampleArticleContentItemLanguageMetadataEnUsWithRelations, ContentItemSamples.SampleArticleContentItemLanguageMetadataEnGbWithRelations,
             ContentItemSamples.SampleArticleWebPageItemWithRelations,
+            ContentItemSamples.SampleArticleWrWebPageAcl,
```

Comment:
What does the abbreviation Wr in SampleArticle**Wr**WebPageAcl and SampleArticle**Wr**WebPageAclMappingModel mean?